nilearn/image/image.py

```diff
@@ -447,7 +447,7 @@ def _pad_array(array, pad_sizes):
                                           upper_paddings,
                                           new_shape)]
 
-    padded[tuple(target_slices)] = array[source_slices].copy()
+    padded[tuple(target_slices)] = array[tuple(source_slices)].copy()
```

Comment:
The `tuple()` fixes address a deprecation that will begin failing in numpy 1.23.